money.go

```diff
@@ -32,9 +35,9 @@ func defaultUnmarshalJSON(m *Money, b []byte) error {
 		return err
 	}
 
-	var amount float64
+	var amount string
```

Comment:
The code looks good but we do it like this we can have conflicts when the official repository updates the code. What about defining a customUnmarshalJSON and using the custom rather than the default?

Reply:
Makes sense! I would add these custom marshal/unmarshal functions inside this library, though. To avoid having every service define the same thing. WDYT?